### PR TITLE
Create script_1.py

### PR DESCRIPTION
**Pull request by AlpsRunner**
Can you  help me?

### CODE REVIEW

**Comment by AlpsRunner** [comment]
good news

**Comment by kurganITteacher** [comment]
ok

**Comment by kurganITteacher** [comment]
good

File: script_1.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# something new
+# and what about you?
```
Comment (kurganITteacher) [inline script_1.py:2]: good idea!

File: script_1.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# something new
```
Comment (kurganITteacher) [inline script_1.py:1]: remove

**Pull request merged**